.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--Read comments, before committing pull request read checklist again
+
+# Checklist:
+
+- [ ] I have performed a self-review of my own code,
+- [ ] I have commented my code, particularly in hard-to-understand areas,
+- [ ] I have made corresponding changes to the documentation,
+- [ ] I have added tests that prove my fix is effective or that my feature works,
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
I think this maybe isnt needed as its automated?

Reply:
Sure. I think we could re-add it if it ever becomes an issue (often changes are committed, the tests are failing and there is a request for review)

Reply:
Resolved in [453f671](https://github.com/geeksforsocialchange/PlaceCal/pull/2630/commits/453f671991dcc3d3ffd32332616652d07ad4d695)